[2025-02-15T01:57:29.645+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data_ETL.transform_alerts manual__2025-02-15T00:56:13.638531+00:00 [queued]>
[2025-02-15T01:57:29.660+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data_ETL.transform_alerts manual__2025-02-15T00:56:13.638531+00:00 [queued]>
[2025-02-15T01:57:29.661+0000] {taskinstance.py:1331} INFO - Starting attempt 6 of 6
[2025-02-15T01:57:29.680+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): transform_alerts> on 2025-02-15 00:56:13.638531+00:00
[2025-02-15T01:57:29.689+0000] {standard_task_runner.py:57} INFO - Started process 138 to run task
[2025-02-15T01:57:29.693+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'weather_data_ETL', 'transform_alerts', 'manual__2025-02-15T00:56:13.638531+00:00', '--job-id', '89', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmpd29tvzug']
[2025-02-15T01:57:29.696+0000] {standard_task_runner.py:85} INFO - Job 89: Subtask transform_alerts
[2025-02-15T01:57:29.812+0000] {task_command.py:410} INFO - Running <TaskInstance: weather_data_ETL.transform_alerts manual__2025-02-15T00:56:13.638531+00:00 [running]> on host cbea98f91475
[2025-02-15T01:57:30.004+0000] {taskinstance.py:1568} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airscholar' AIRFLOW_CTX_DAG_ID='weather_data_ETL' AIRFLOW_CTX_TASK_ID='transform_alerts' AIRFLOW_CTX_EXECUTION_DATE='2025-02-15T00:56:13.638531+00:00' AIRFLOW_CTX_TRY_NUMBER='6' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-15T00:56:13.638531+00:00'
[2025-02-15T01:57:30.165+0000] {python.py:183} INFO - Done. Returned value was: None
[2025-02-15T01:57:30.193+0000] {taskinstance.py:1368} INFO - Marking task as SUCCESS. dag_id=weather_data_ETL, task_id=transform_alerts, execution_date=20250215T005613, start_date=20250215T015729, end_date=20250215T015730
[2025-02-15T01:57:30.229+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2025-02-15T01:57:30.277+0000] {taskinstance.py:2674} INFO - 0 downstream tasks scheduled from follow-on schedule check
